packages/gasket-plugin-docusaurus/test/docusaurus.test.config.js

```diff
@@ -0,0 +1,23 @@
+module.exports = {
```

Comment:
For "fixtures" like this, our pattern has been to place under a `./test/fixtures` directory to signify their purpose. Also, you could probably remove `.test` from the name to avoid it being confused a file containing tests.